tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -956,15 +956,24 @@
         } else {
           // Selecting the node triggers a pan to it.
           const nodeMap = graph.get('renderHierarchy').hierarchy.getNodeMap();
-          if (!nodeMap[nodeName]) {
-            // In some cases, a node with other names using its name as
-            // name scope may not have base-expanded (i.e., normalized) name
-            // in the graph visualizer. In that case, we use the non-expanded
-            // node name.
+          if (nodeMap[nodeName] == null) {
+            // Some nodes may not be present in the graph scene, due to either
+            // of the two reasons:
+            //   1. A node with other names using its name as name scope may
+            //      not have a base-expanded (i.e., normalized) name in the
+            //      graph visualizer.
+            //   2. The node may fall into a type of nodes automatically
+            //      omitted by the graph visualizer, e.g., certain nodes
+            //      related to TensorBoard summaries.
+            // To deal with case 1, we use the non-expanded node name.
+            // To deal with case 2, we check whether `nodeMap[nodeName]` equal
+            // `null` again before setting `selectedNode` below.
             nodeName =
                 tf_debugger_dashboard.removeNodeNameBaseExpansion(nodeName);
           }
-          graph.set('selectedNode', nodeName);
+          if (nodeMap[nodeName] != null) {
```

Comment:
Maybe just make into an else statement.

Reply:
Thanks for the review! 

Regarding this comment, I don't think `else if` or `else` will work here. If the condition on line 959 is true, we want lines 971-972 to execute, then we want to check the result of the execution again on this line. If 'else` or `else if` is used, the second check will not happen.

Reply:
Ah, indeed!